docs/appendix/glossary/board.md

```diff
@@ -2,11 +2,11 @@
 title: Board
 id: board
 full_link: /components/board/
-short_description: A board is the signal wire hub of a machine that provides access to GPIO pins, SPI or I2C.
+short_description: A board is the signal wire hub of a machine that provides access to GPIO pins.
 aka:
 ---
 
-A board is the signal wire hub of a machine that provides access to GPIO pins, SPI or I2C.
+A board is the signal wire hub of a machine that provides access to GPIO pins.
```

Comment:
Off-topic question: what's a single wire hub? Boards have multiple wires, which makes me think I've misunderstood. Should that part be rephrased? Same (still off-topic) question on line 5.

Reply:
:facepalm: I misread "signal" as "single" like 3 times. This sentence is fine as-is!